python/cuml/test/dask/test_kneighbors_classifier.py

```diff
@@ -108,9 +108,9 @@ def check_probabilities(l_probas, d_probas):
 
 
 @pytest.mark.parametrize("datatype", ['dask_array', 'dask_cudf'])
-@pytest.mark.parametrize("n_neighbors", [1, 3, 8])
-@pytest.mark.parametrize("n_parts", [2, 4, 12])
-@pytest.mark.parametrize("batch_size", [128, 1024])
+@pytest.mark.parametrize("n_neighbors", [8])
```

Comment:
Testing the degrenerate n_neighbors=1 case to me seems important. I always like to test the extremes. What about doing mark_parametrize with a list of tuples, like:
``` # neighbors,parts,batch_size
[(1, 3, 256),
 (8, 8, 256),
 (9, 3, 128)]
```

Then you still only run 3 times but still examine the extremes